docs/en/siem/installation.asciidoc

```diff
@@ -30,13 +31,77 @@ for sorting. For more information, see {kib}.
 [[install-beats]]
 === Install {beats} shippers
 
-To populate the security UI with metrics and
-log data, you need to install and configure the following shippers:
+To populate the {siem-ui} with hosts and network data, you need to install and
+configure Beats on the endpoints you want to collect and send data:
 
-* https://www.elastic.co/products/beats/packetbeat[{packetbeat}] for analyzing
-network packets 
 * https://www.elastic.co/products/beats/filebeat[{filebeat}] for forwarding and
 centralizing logs and files
 * https://www.elastic.co/products/beats/auditbeat[{auditbeat}] for monitoring
 directories for file changes
+* https://www.elastic.co/products/beats/winlogbeat[{winlogbeat}] for centralizing logs
+* https://www.elastic.co/products/beats/packetbeat[{packetbeat}] for analyzing
+network packets 
+
+NOTE: {siem-soln} also works with custom and third-party data sources in addition to Beats.
+{ecs-ref}[Elastic Common Schema (ECS)] makes this possible. Any data tagged as `host.name`
+can be used in {siem-soln}. 
```

Comment:
I would leave out the `host.name` mention here. That's part of ECS and I think we shouldn't highlight it more than other ECS fields, in this context.

Reply:
🤔 Is `host.name` still the main key by which hosts are identified (vs `host.id` and `host.hostname`) in the SIEM app? Perhaps we should actually expand on this fact here.

Reply:
Hi @webmat - Karen removed this as per Tudor's suggestion.

I think we should expand on host.name vs host.hostname in the ECS reference guide. What do you think?